### PR TITLE
setup frontend

### PR DESCRIPTION
**Pull request by tauqeer92**

### CODE REVIEW

**Comment by arhussain1** [comment]
Hey Tauqeer, great work on creating your first PR. Some initial advice, when creating a PR ensure you have a description of:
- what feature or task you hope to complete with this PR
- would also be great to add any specific areas of code which you would like for reviewers to pay extra attention to

Going to take a more detailed look at this PR now

**Comment by arhussain1** [comment]
I noticed a few additional files that could be removed such as:
- favIcon.ico
- the logo png files
- manifest.json (since we are not using the icon and png files above)
- robots.txt file can be removed as we don't expect for this app to be commercial and we don't care how engine crawlers see it (if you're confused by what I just said let me know and we can go into greater detail)

For the src folder just keep the following:
- App.css (remove all the unused css since we removed some stuff)
- App.js (remove unused dependencies)
- index.css
- index.js (remove unused dependencies)

Any questions on the above let me know 😆

**Review by arhussain1 — CHANGES_REQUESTED** [review]
I left some comments for some files that are not needed atm

**Comment by tauqeer92** [comment]
> Hey Tauqeer, great work on creating your first PR. Some initial advice, when creating a PR ensure you have a description of:
> 
> * what you feature or task you hope to complete with this PR
> * would also be great to add any specific areas of code which you would like for reviewers to pay extra attention to
> 
> Going to take a more detailed look at this PR now



> I noticed a few additional files that could be removed such as:
> 
> * favIcon.ico
> * the logo png files
> * manifest.json (since we are not using the icon and png files above)
> * robots.txt file can be removed as we don't expect for this app to be commercial and we don't care how engine crawlers see it (if you're confused by what I just said let me know and we can go into greater detail)
> 
> For the src folder just keep the following:
> 
> * App.css (remove all the unused css since we removed some stuff)
> * App.js (remove unused dependencies)
> * index.css
> * index.js (remove unused dependencies)
> 
> Any questions on the above let me know 😆
Thanks for the feedback, I was planning on removing the files as I develop the frontend so I can see the importance of each file and how it connects while I'm developing compared to just deleting it in the beginning without know its true importance

**Comment by arhussain1** [comment]
You can definitely go about it this way, but I would prefer it if you could remove the files while you have the opportunity in this PR

It is always good practice to remove these unnecessary files early on, this allows us to move forward with a clean repo without additional file dependencies.

With regards to
> Thanks for the feedback, I was planning on removing the files as I develop the frontend so I can see the importance of each file and how it connects while I'm developing compared to just deleting it in the beginning without know its true importance

I have a couple of tips for you:
1. Remove any obvious files that you know will not be required
2. Run this locally and see if anything breaks, follow the stack trace and find the file it's referring to
3. Remove these dependencies that are causing the app to break (as you don't need them anymore)

If there are any files in the previous comment that are confusing let me know.

**Comment by tauqeer92** [comment]
I've removed the unnecessary files

File: frontend/src/App.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
```
Comment (arhussain1) [inline frontend/src/App.css:5]: You could probably remove some of this css since we remove the logo and a few other elements

**Comment by arhussain1** [comment]
Not going to block the PR, but there are a few css styles that can be removed as they are not being used anymore

**Review by arhussain1 — APPROVED** [review]
Some of the css is unused and should be removed but you can leave that for another PR

**Pull request merged**